Source/x11/XGServer.m

```diff
@@ -819,6 +821,9 @@ - (BOOL) setPreeditSpot: (NSPoint *)p
 #include <AppKit/NSClipView.h>
 #include <AppKit/NSTextView.h>
 
+#pragma clang diagnostic push
```

Comment:
This will generate useless warnings for gcc. Either make it conditional on clang or remove it again.

Reply:
Good point. Fixed.